cmd/fleet/cron.go

```diff
@@ -1173,3 +1174,65 @@ func stringSliceToUintSlice(s []string, logger kitlog.Logger) []uint {
 	}
 	return result
 }
+
+// newIPhoneIPadRefetcher will enqueue DeviceInformation commands on iOS/iPadOS devices
+// to refetch their host details.
+//
+// See https://developer.apple.com/documentation/devicemanagement/get_device_information.
+//
+// We will refetch iPhones/iPads every 1 hour (to match the default
+// detail interval of all (osquery-capable) hosts in Fleet).
+func newIPhoneIPadRefetcher(
```

Comment:
Nit, but if we eventually add e.g. Apple Watch and future Apple stuff, should we use `IDevice`? (or something even less associated with i-naming?)